server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -470,6 +472,19 @@ protected void doMerge(final ObjectMapper mergeWith) {
         }
     }
 
+    private static void checkEnabledFieldChange(ObjectMapper mergeWith, Mapper mergeWithMapper, Mapper mergeIntoMapper) {
+        if (mergeIntoMapper instanceof ObjectMapper && mergeWithMapper instanceof ObjectMapper) {
+            final ObjectMapper mergeIntoObjectMapper = (ObjectMapper) mergeIntoMapper;
+            final ObjectMapper mergeWithObjectMapper = (ObjectMapper) mergeWithMapper;
+
+            if (mergeIntoObjectMapper.isEnabled() != mergeWithObjectMapper.isEnabled()) {
+                final String path = mergeWith.fullPath() + "." + mergeWithObjectMapper.simpleName() + ".enabled";
+
```

Comment:
nit: maybe delete this line

Reply:
Yes, these two lines are related, no need to add a blank line, I'll remove it.